bin/scripts/gotta-patch-em-all-font-patcher!.sh

```diff
@@ -62,12 +62,18 @@ echo "$LINE_PREFIX Total source fonts found: ${#source_fonts[*]}"
 function patch_font {
   local f=$1; shift
   local i=$1; shift
+  local purge=$1; shift
   # take everything before the last slash (/) to start building the full path
   local patched_font_dir="${f%/*}/"
   # find replace unpatched parent dir with patched parent dir:
   local patched_font_dir="${patched_font_dir/$unpatched_parent_dir/$patched_parent_dir}"
 
   [[ -d "$patched_font_dir" ]] || mkdir -p "$patched_font_dir"
+  if [ -n ${purge} -a -d "${patched_font_dir}complete" ]
+  then
+    echo "Purging patched font dir ${patched_font_dir}complete"
+    rm ${patched_font_dir}complete/*.[to]tf
```

Comment:
It does not purge out `readme`s or any other files.
But maybe we want to do that (purge out **all** files), so that the `patched-fonts/` dir just contains all autogenerated content.

If so we can change the new line 75 accordingly.

On the other hand, in that case the workflow could just wipe the `patched-fonts/` dir before calling `patch-em-all`; and we do not need any of these changes (i.e. this PR)
The drawback of THAT solution is, that it fixes it for our github workflow, but not for any self patching user.